### PR TITLE
Migrate PDF URNs with trailing #

### PR DESCRIPTION
**Pull request by seanh**
This depends on <https://github.com/hypothesis/h/pull/3525> (one commit from #3525 is cherry-picked into this branch)

### CODE REVIEW

**Comment by seanh** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;p6wuB946&#x2F;382-migrate-pdf-urns-with-trailing-created-by-safari"> Migrate PDF URNs with trailing &#39;#&#39; created by Safari</a>

**Comment by seanh** [comment]
Still need to update `target_uri`

**Comment by seanh** [comment]
Done

**Comment by seanh** [comment]
Ready, but depends on https://github.com/hypothesis/h/pull/3525

**Comment by nickstenning** [comment]
As mentioned in 6d4eb90, migrations shouldn't be importing from `models` modules. If you can fix that up we can get this merged.

**Comment by seanh** [comment]
Seems to work

File: h/migrations/versions/9e6b4f70f588_removing_trailing_from_pdf_urns.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+class DocumentURI(Base):
+    __tablename__ = 'document_uri'
+    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+    claimant = sa.Column('claimant', sa.UnicodeText, nullable=False)
+    claimant_normalized = sa.Column('claimant_normalized',
+                                    sa.UnicodeText,
+                                    nullable=False)
+    uri = sa.Column('uri', sa.UnicodeText, nullable=False)
+    uri_normalized = sa.Column('uri_normalized',
+                               sa.UnicodeText,
```
Comment (nickstenning) [inline h/migrations/versions/9e6b4f70f588_removing_trailing_from_pdf_urns.py]: It doesn't matter that much but two things to note:

1. There's no need to specify the column name twice when using `declarative_base`: `uri = sa.Column(sa.UnicodeText, )` will work fine.
2. Usually in migrations you don't need to specify all the column properties, just the types, so sqla knows how to map resultsets onto these objects.

**Comment by nickstenning** [comment]
Alrighty. LGTM.

**Pull request merged**